FlowCrypt/Functionality/Mail Provider/MessagesList Provider/Model/Message.swift

```diff
@@ -14,6 +14,7 @@ struct Message: Hashable {
     let identifier: Identifier
     let date: Date
     let sender: String?
+    let receiver: String?
```

Comment:
For consistency it'll be better to rename `receiver` to `recipient`, as we already have `recipient` in other parts of the app